.github/ISSUE_TEMPLATE/feature_request.yaml

```diff
@@ -0,0 +1,52 @@
+name: Feature request
+description: Suggest an idea for this project
+title: "[Feature] <title>"
+labels: "feature"
+
+body:
+  - type: markdown
+    attributes:
+      value: Is your feature request related to a problem? Please describe.
+  - type: checkboxes
+    attributes:
+      label: Choice Programming Language
+      description: Plz choose Programming Language what you want to use Euphony
+      options:
+        - label: C++
+        - label: JAVA
+        - label: javaScript
+        - label: Kotlin
+        - label: Other
```

Comment:
I don't think the programming language matters. 
If the feature is implemented in any language, it is not difficult to port it, so I think this may not be necessary :)

Reply:
Should I delete this code? 😃